src/app/parcels/batch/BatchParcelDataGrid.test.tsx

```diff
@@ -0,0 +1,15 @@
+import BatchParcelDataGrid from "@/app/parcels/batch/BatchParcelDataGrid";
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { expect, it } from "@jest/globals";
+import "@testing-library/jest-dom/jest-globals";
+
+describe("Parcels - Batch Parcel Data Grid", () => {
+    beforeEach(() => {
+        render(<BatchParcelDataGrid />);
+    });
+    it("renders intial header view", () => {
+        expect(screen.getByText("Full Name")).toBeInTheDocument();
+        expect(screen.getByText("Phone Number")).toBeInTheDocument();
+    });
+});
```

Comment:
We can be a lot more thorough with these tests.

Firstly, we can export the columns definition from the main file (preferably with a more descriptive name than "columns"), and check that all of the column headings are rendered. This will not only provide better coverage, but make it more maintainable since we won't have to update the test if the columns change.

Secondly, you'll need a test to make sure all the data is rendered correctly. With static data this is probably best as a snapshot test